### ISSUE TITLE
Where is the sigmoid function?

### ISSUE THREAD

**Issue by Psilym** [open-mmlab/mmsegmentation#3495]
I tried PSPNet in a 2D segmentation task. The loss config has specified DiceLoss('use_sigmoid = True'). The network only uses ReLu to activate. However, during evaluation process, the results are automatically normalized to (0,1) probabilities. Thereby, I wonder where the sigmoid function is utilized in the mmseg structure during prediction(or inference) stage. 
To be specified, during prediction/inference, where is the sigmoid from non-normalized **seg_logits** to normalized prediction probability?
ps: Besides, I find the structure is hard to locate and find out the actual sequence of code running. Is there a good solution?

**Comment by Psilym** [comment]
Does anyone know the location of sigmoid?

**Comment by AI-Tianlong** [comment]
So you mean, in predict, seg_logits = (0,1)? Or is the final mask {0,1} of these two classes?

**Comment by Psilym** [comment]
> So you mean, in predict, seg_logits = (0,1)? Or is the final mask {0,1} of these two classes?

Thanks for response. The `seg_logits` are non-normalized to [0,1]. But where is `seg_logits` transformed to normalized values `pred_sem_seg` in mmseg architecture?

**Comment by Psilym** [comment]
I have found the transforming from non-normalized `seg_logits` to normalized `pred_sem_seg`. The code is in `models/segmentors/base.py`->`postprocess_result`function, which can be self-reloaded.
